test/dx-new-request.js

```diff
@@ -0,0 +1,258 @@
+'use strict';
+const config = require('config');
+const client = require('./lib/client').client;
+const nav = require('./lib/nav/navigation')(client, config);
+const micis = require('./lib/auth/micis')(client);
+const should = require('should');
+
+describe('Test data exchange functionality for a new request', function() {
+  this.timeout(config.defaultTimeout);
+
+  before('initialize', function(done) {
+    client.clientReady.then(function boot() {
+      if (!micis.loggedOn) {
+        micis.logon();
+      }
+
+      client.call(done);
+    });
+  });
+
+  it('should open Data Exchange', function(done) {
+    nav
+      .micisMenu
+      .clickNested('Browse Available Data')
+      .call(done);
+  });
+
+  it('should verify that [Previous Request Templates] have loaded', function(done) {
+    const previousRequestTemplates = '#requestMenu > option';
+
+    client
+      .elements(previousRequestTemplates)
+      .then(response => {
+        response.value.length.should.be.greaterThan(1);
+      })
+      .call(done);
+  });
+
+  it('should verify that all 4 modality filters have loaded', function(done) {
```

Comment:
Same thing as the previous comment: isn’t this already tested in _test/dx-clone.js_?